kernel/src/process_manager/process_paging.rs

```diff
@@ -286,6 +286,12 @@ impl ProcessPageTableRef {
         self.add_region_vaddr(VirtAddr::from(0x18000000000u64), data);
     }
 
+    pub fn add_function(&self, data:VirtAddr, size: u64) {
+        let data: *mut u8 = data.as_mut_ptr::<u8>();
+        let data = unsafe { slice::from_raw_parts(data, size as usize) };
+        self.add_region_vaddr(VirtAddr::from(0xFE8000000000u64), data);
```

Comment:
If I changed the address to, say, 
```rust
self.add_region_vaddr(VirtAddr::from(0x140_0000_0000u64), data);
```
then I confirmed that
```
[libos]
entrypoint = "/lib/python"

[loader]
argv = ["python3", "-c", "print('import ctypes'); import ctypes; print('string_at'); a=ctypes.string_at(0x14000000000); print(a)"]
...
```

this manifest works.

Reply:
```
self.add_region_vaddr(VirtAddr::from(0x7F80_0000_0000u64), data);
```
is ok but
```
self.add_region_vaddr(VirtAddr::from(0x8000_0000_0000u64), data);
```
is not (PGD idx=256)

Reply:
ok this is because
> Address sizes:                      43 bits physical, 48 bits virtual